sd-step.go

```diff
@@ -69,7 +69,7 @@ func execHab(pkgName string, pkgVersion string, command []string, output io.Writ
 		return verErr
 	}
 
-	installCmd := []string{habPath, "pkg", "install", pkg}
+	installCmd := []string{habPath, "pkg", "install", pkg, "1>/dev/null"}
```

Comment:
The intention here is to only redirect the STDOUT? Then we do not need the 1. By default `>/dev/null` this is redirecting the STDOUT right?